test/cql-pytest/cassandra_tests/validation/operations/compact_storage_test.py

```diff
@@ -2584,19 +2571,6 @@ def testAllowFilteringOnPartitionKeyWithDistinct(cql, test_keyspace):
             assert_rows(execute(cql, table, "SELECT DISTINCT pk0, pk1 FROM %s WHERE pk1 > 1 LIMIT 3 ALLOW FILTERING"),
                        row(2, 2))
 
-    # Test a 'wide row' thrift table.
-    with create_table(cql, test_keyspace, "(pk int, name text, val int, PRIMARY KEY (pk, name)) WITH COMPACT STORAGE") as table:
```

Comment:
Again, please don't remove this test (or any test, unless it really tests the Thrift protocol).

Reply:
ack. will also revert this change.